<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormLogftp</class>
 <widget class="QWidget" name="FormLogftp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>843</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,3">
   <item row="0" column="0">
    <widget class="QTreeWidget" name="treeWidget">
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="0" column="1" rowspan="2">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout" columnstretch="1,1,1">
      <item row="0" column="0" rowspan="5">
       <widget class="QTreeWidget" name="treeWidget_log">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="autoExpandDelay">
         <number>0</number>
        </property>
        <column>
         <property name="text">
          <string>日志名称</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>日志描述</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>日志大小</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>修改时间</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_searchStr">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QToolButton" name="pushButton_Search">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>搜索</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QTextEdit" name="textEdit_log"/>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QLabel" name="label_warings">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>欢迎使用维护软件</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="2">
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QToolButton" name="tb_connect">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>连接</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QToolButton" name="tb_close">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>断开</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
